brainstormforce/ultimate-addons-for-gutenberg
# WIP

### Description

Added Anchor support to UAG blocks.

### Screenshots


### Types of changes




New feature - Added Anchor compatibility to blocks.

### How has this been tested?

- Add an anchor tag to block
- Apply that anchor tag to any of heading or block
- Click on a heading or block check it directed or not
- Another one inspecting and checking is the anchor tag id added to block or not.

### Checklist:
